lametro/static/css/city_custom.css

```diff
@@ -331,21 +331,6 @@ hr .events-line {
     }
 }
 
-@media (min-width: 767px) and (max-width: 1180px) {
-    .navbar-collapse.collapse {
-        display: none !important;
-    }
-    .navbar-collapse.collapse.in {
-        display: block !important;
-    }
-    .navbar-header .collapse, .navbar-toggle {
-        display:block !important;
-    }
-    .navbar-header {
-        float:none;
-    }
-}
-
```

Comment:
curious if you looked to see where else these classes were being used // if elements affected by these had their behavior changed by removing this styling?

Reply:
I believe this block was just used to set the breakpoint for the navbar at a larger window size than bootstrap's default. Most likely because the navbar had so much on it.

It's a media rule plus only the navbar is styled here so I don't think anything else is affected. But if you saw any weird behavior, let me know 🙏🏾

Reply:
@fatima3558 Anything you want me to modify before I get your approval?